packages/hint-no-vulnerable-javascript-libraries/tests/tests.ts

```diff
@@ -26,6 +26,10 @@ const defaultTests: HintTest[] = [
     {
         name: `page with a vulnerable library (jquery 2.1.4) fails`,
         reports: [{
+            documentation: [{
+                link: 'https://snyk.io/vuln/npm:jquery',
```

Comment:
By chance do we have access to direct links to each of the vulnerabilities? If so, I think we should link to the page for each directly (with link text referencing the vulnerability id/name) instead of the all-up page for the library. Good opportunity to potentially list multiple documentation links.